### PR TITLE
Add canonical data for palindrome products

### PR DESCRIPTION
**Pull request by stevejb71**

### CODE REVIEW

**Comment by Insti** [comment]
I have no recent experience with this problem, but this file looks reasonable. 👍

File: exercises/palindrome-products/canonical-data.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    [1, 9],
+                    [3, 3]
+                ]
+            }
+        },
+        {
+            "description": "find the largest palindrome from double digit factors",
+            "property": "largest",
+            "input_min": 10,
+            "input_max": 99,
```
Comment (Insti) [inline exercises/palindrome-products/canonical-data.json]: Missing test for the smallest in this range?
It is used as an example in the description, so I would expect to find it here.

Comment (stevejb71) [reply exercises/palindrome-products/canonical-data.json]: I added this test.

**Comment by petertseng** [comment]
Please say that this closes
https://github.com/exercism/problem-specifications/issues/573

**Pull request merged**